cmd/package.go

```diff
@@ -192,16 +217,36 @@ func (b *PackageHelper) PackageWindows(po *ProjectOptions, cleanUp bool) error {
 	// Build syso
 	sysofile := filepath.Join(b.fs.Cwd(), basename+"-res.syso")
 
-	batfile, err := fs.LocalDir(".")
-	if err != nil {
-		return err
-	}
+	// cross-compile
+	if b.platform != runtime.GOOS {
+		folder, err := os.Getwd()
+		if err != nil {
+			return err
+		}
 
-	windresBatFile := filepath.Join(batfile.fullPath, "windres.bat")
-	windresCommand := []string{windresBatFile, sysofile, tgtRCFile}
-	err = NewProgramHelper().RunCommandArray(windresCommand)
-	if err != nil {
-		return err
+		args := []string{
+			"docker", "run", "--rm",
+			"-v", folder + ":/build",
+			"--entrypoint", "/bin/sh",
+			"techknowlogick/xgo",
+			"-c", "/usr/bin/x86_64-w64-mingw32-windres -o /build/" + basename + "-res.syso /build/" + basename + ".rc",
+		}
+
+		if err := NewProgramHelper().RunCommandArray(args); err != nil {
+			return err
```

Comment:
What might be nice is to report to the user that docker doesn't exist and to install it, but I'm guessing the default message would be something similar to that...

Reply:
same as if xgo is not installed, they should get a command not found error.
There is already a check for mewn, so adding the 2 checks (xgo/docker) shouldnt take much time, let me know if I've to add them in this PR or a new once (after this one is merged).

I was thinking also on using the `builds` folder as a standard directory where to put both cross-compiled and normal builds, less mess on project folder.

Reply:
I would recommend adding them to this PR.  I don't see any reason you shouldn't `fixup` the commit into the one on this branch to resolve this conversation.